### ISSUE TITLE
Error on server startup

### ISSUE THREAD

**Issue by nmurphy101** [ssblur/scriptor#18]
Error's out on server start
```
[00:47:27] [main/INFO]: Loading Minecraft 1.19.2 with Fabric Loader 0.14.17
[00:47:27] [main/INFO]: Loading 43 mods:
	- architectury 6.5.69
	- fabric-api 0.75.1+1.19.2
	   |-- fabric-api-base 0.4.15+8f4e8eb390
	   |-- fabric-api-lookup-api-v1 1.6.14+93d8cb8290
	   |-- fabric-biome-api-v1 9.1.0+82db024a90
	   |-- fabric-block-api-v1 1.0.2+e415d50e90
	   |-- fabric-blockrenderlayer-v1 1.1.25+cafc6e8e90
	   |-- fabric-client-tags-api-v1 1.0.5+b35fea8390
	   |-- fabric-command-api-v1 1.2.16+f71b366f90
	   |-- fabric-command-api-v2 2.2.1+413cbbc790
	   |-- fabric-commands-v0 0.2.33+df3654b390
	   |-- fabric-containers-v0 0.1.41+df3654b390
	   |-- fabric-content-registries-v0 3.5.1+7c6cd14d90
	   |-- fabric-convention-tags-v1 1.2.2+27265b3490
	   |-- fabric-crash-report-info-v1 0.2.8+aeb40ebe90
	   |-- fabric-data-generation-api-v1 5.3.7+413cbbc790
	   |-- fabric-dimensions-v1 2.1.35+0d0f210290
	   |-- fabric-entity-events-v1 1.5.4+9244241690
	   |-- fabric-events-interaction-v0 0.4.34+562bff6e90
	   |-- fabric-events-lifecycle-v0 0.2.35+df3654b390
	   |-- fabric-game-rule-api-v1 1.0.24+b6b6abb490
	   |-- fabric-item-api-v1 1.6.5+dce4603e90
	   |-- fabric-item-groups-v0 0.3.38+9244241690
	   |-- fabric-key-binding-api-v1 1.0.25+5c4fce2890
	   |-- fabric-keybindings-v0 0.2.23+df3654b390
	   |-- fabric-lifecycle-events-v1 2.2.4+1b46dc7890
	   |-- fabric-loot-api-v2 1.1.12+83a8659290
	   |-- fabric-loot-tables-v1 1.1.15+9e7660c690
	   |-- fabric-message-api-v1 5.0.7+93d8cb8290
	   |-- fabric-mining-level-api-v1 2.1.23+33fbc73890
	   |-- fabric-models-v0 0.3.21+c6af733c90
	   |-- fabric-networking-api-v1 1.2.11+10eb22f490
	   |-- fabric-networking-v0 0.3.28+df3654b390
	   |-- fabric-object-builder-api-v1 4.2.1+d8ef690890
	   |-- fabric-particles-v1 1.0.14+4d0d570390
	   |-- fabric-recipe-api-v1 1.0.1+413cbbc790
	   |-- fabric-registry-sync-v0 0.9.32+9244241690
	   |-- fabric-renderer-api-v1 1.2.1+1adbf27790
	   |-- fabric-renderer-indigo 0.8.0+1adbf27790
	   |-- fabric-renderer-registries-v1 3.2.24+df3654b390
	   |-- fabric-rendering-data-attachment-v1 0.3.19+6e0787e690
	   |-- fabric-rendering-fluids-v1 3.0.11+4d0d570390
	   |-- fabric-rendering-v0 1.1.27+df3654b390
	   |-- fabric-rendering-v1 1.12.1+d8ef690890
	   |-- fabric-resource-conditions-api-v1 2.1.2+aae9039d90
	   |-- fabric-resource-loader-v0 0.8.3+251d254a90
	   |-- fabric-screen-api-v1 1.0.32+4d0d570390
	   |-- fabric-screen-handler-api-v1 1.3.7+1cc24b1b90
	   |-- fabric-sound-api-v1 1.0.2+c4f28df590
	   |-- fabric-textures-v0 1.0.24+aeb40ebe90
	   |-- fabric-transfer-api-v1 2.1.6+413cbbc790
	   \-- fabric-transitive-access-wideners-v1 1.3.3+08b73de490
	- fabricloader 0.14.17
	- java 17
	- minecraft 1.19.2
	- scriptor 0.2.12
[00:47:27] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/F:/Downloads/Server%20Files/libraries/net/fabricmc/sponge-mixin/0.12.4+mixin.0.8.5/sponge-mixin-0.12.4+mixin.0.8.5.jar Service=Knot/Fabric Env=SERVER
[00:47:27] [main/INFO]: Compatibility level set to JAVA_16
[00:47:27] [main/INFO]: Compatibility level set to JAVA_17
[00:47:31] [main/INFO]: Building unoptimized datafixer
[00:47:32] [main/ERROR]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'scriptor'!
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51) ~[fabric-loader-0.14.17.jar:?]
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33) ~[fabric-loader-0.14.17.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49) ~[fabric-loader-0.14.17.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35) ~[fabric-loader-0.14.17.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startServer(Hooks.java:62) ~[fabric-loader-0.14.17.jar:?]
	at net.minecraft.server.Main.main(Main.java:101) [server-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462) [fabric-loader-0.14.17.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.17.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotServer.main(KnotServer.java:23) [fabric-loader-0.14.17.jar:?]
	at net.fabricmc.loader.impl.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:69) [fabric-loader-0.14.17.jar:?]
Caused by: java.lang.NoClassDefFoundError: net/minecraft/class_5614
	at com.ssblur.scriptor.blockentity.ScriptorBlockEntities.registerRenderers(ScriptorBlockEntities.java:30) ~[scriptor-0.2.12.jar:?]
	at com.ssblur.scriptor.blockentity.ScriptorBlockEntities.register(ScriptorBlockEntities.java:26) ~[scriptor-0.2.12.jar:?]
	at com.ssblur.scriptor.ScriptorMod.init(ScriptorMod.java:37) ~[scriptor-0.2.12.jar:?]
	at com.ssblur.scriptor.fabriclike.ScriptorModFabricLike.init(ScriptorModFabricLike.java:7) ~[scriptor-0.2.12.jar:?]
	at com.ssblur.scriptor.fabric.ScriptorModFabric.onInitialize(ScriptorModFabric.java:9) ~[scriptor-0.2.12.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.14.17.jar:?]
	... 7 more
Caused by: java.lang.ClassNotFoundException: net.minecraft.class_5614
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226) ~[fabric-loader-0.14.17.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:112) ~[fabric-loader-0.14.17.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?]
	at com.ssblur.scriptor.blockentity.ScriptorBlockEntities.registerRenderers(ScriptorBlockEntities.java:30) ~[scriptor-0.2.12.jar:?]
	at com.ssblur.scriptor.blockentity.ScriptorBlockEntities.register(ScriptorBlockEntities.java:26) ~[scriptor-0.2.12.jar:?]
	at com.ssblur.scriptor.ScriptorMod.init(ScriptorMod.java:37) ~[scriptor-0.2.12.jar:?]
	at com.ssblur.scriptor.fabriclike.ScriptorModFabricLike.init(ScriptorModFabricLike.java:7) ~[scriptor-0.2.12.jar:?]
	at com.ssblur.scriptor.fabric.ScriptorModFabric.onInitialize(ScriptorModFabric.java:9) ~[scriptor-0.2.12.jar:?]
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47) ~[fabric-loader-0.14.17.jar:?]
	... 7 more

```

**Comment by ssblur** [comment]
Thank you for reporting. Able to replicate, need to limit render registry functions to the client. Testing a fix at the moment, will push an update as soon as I've verified it's working.

**Comment by ssblur** [comment]
Fixed by 1446f09

**Comment by nmurphy101** [comment]
Nice. :)
